package/display/package

```diff
@@ -114,6 +118,7 @@ rm2fb-client() {
             install -D -m 644 -t "$pkgdir"/etc/systemd/system/draft.service.d "$srcdir"/rm2fb-preload.conf
             install -D -m 644 -t "$pkgdir"/etc/systemd/system/remux.service.d "$srcdir"/rm2fb-preload.conf
             install -D -m 644 -t "$pkgdir"/etc/systemd/system/tarnish.service.d "$srcdir"/rm2fb-preload.conf
+            install -D -m 644 -t "$pkgdir"/lib/udev/rules.d/ "$srcdir"/15-systemd-input.rules
```

Comment:
Why add the udev rule to both the display and the rm2fb-client packages?

Reply:
I was a bit confused about this. I thought one is for rm1 and other is for rm2. my intention is to include this rule in rm1/rm2 in display package